### PR TITLE
Validation add missing descriptions

### PR DESCRIPTION
**Pull request by chrisms86**

### CODE REVIEW

File: docs/outlook/mapi/pidtagincompletecopy-canonical-property.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "PidTagIncompleteCopy Canonical Property"
- 
- 
+description: This article outlines the PidTagIncompleteCopy canonical property, which contains TRUE if a message is an incomplete copy of another message.
```
Comment (MeeraDi) [inline docs/outlook/mapi/pidtagincompletecopy-canonical-property.md:3]: For consistency, please use either "This article outlines" or "Outlines"

File: docs/outlook/mapi/pidtaginconflict-canonical-property.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "PidTagInConflict Canonical Property"
- 
- 
+description: This article outlines the PidTagInConflict canonical property, which contains TRUE when an attachment represents an alternate replica.
```
Comment (dereklh7) [inline docs/outlook/mapi/pidtaginconflict-canonical-property.md:3]: Remove "This article".

File: docs/outlook/mapi/pidtaginitials-canonical-property.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "PidTagInitials Canonical Property"
- 
- 
+description: This article outlines the PidTagInitials canonical property, which contains the initials for parts of the full name of the recipient.
```
Comment (dereklh7) [inline docs/outlook/mapi/pidtaginitials-canonical-property.md:3]: Remove "This article".

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**
@lindalu-MSFT 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**